.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = testscpp/Catch2
 	url = https://github.com/catchorg/Catch2.git
 	branch = v2.x
+[submodule "extern/taskflow"]
+	path = extern/taskflow
+	url = https://github.com/taskflow/taskflow.git
```

Comment:
IMO we should point to a specific tagged release version, rather than just the main branch

Reply:
Current latest?

Reply:
Assuming I understand your question correctly, whatever the most recent tag is would make sense to me.

Reply:
v3.4.0: https://github.com/taskflow/taskflow/tags

Reply:
Ah, I think I meant to make this comment on https://github.com/dwavesystems/dwave-preprocessing/pull/71, though perhaps that was also pointing to a tag and I just misunderstood.

Reply:
It seems like there is still no elegant way to specify a tag/commit for a submodule so I'm explicitly checked out the commit for that tag and committed it.  This is how I've done it in the past and I don't know a better way.

Reply:
That makes sense. And putting it in the commit message like you have also makes sense.